plugins/module_utils/azure_rm_common.py

```diff
@@ -1388,10 +1406,14 @@ def _get_msi_credentials(self, subscription_id_param=None, **kwargs):
             'subscription_id': subscription_id
         }
 
-    def _get_azure_cli_credentials(self, resource=None):
+
```

Comment:
There is a line between functions (there are two blank lines)。

Reply:
fixed